<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="PORTS_info_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PORTS INFO</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="PWM_A_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>PWM A</string>
    </property>
    <widget class="QCheckBox" name="PWM_A_ON_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PWM_A_duty_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Duty:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SEND_data_button_A">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send data</string>
     </property>
    </widget>
    <zorder>SEND_data_button_A</zorder>
    <zorder>PWM_A_ON_checkBox</zorder>
    <zorder>PWM_A_duty_edit</zorder>
    <zorder>label_2</zorder>
   </widget>
   <widget class="QGroupBox" name="PWM_B_groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>PWM B</string>
    </property>
    <widget class="QCheckBox" name="PWM_B_ON_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PWM_B_duty_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Duty:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SEND_data_button_B">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send data</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="COM_port_select">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM20</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="PORTS_info_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BAUD_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>9600</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BAUD:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DATA_bits_select">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>5 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 bit</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="STOP_bits_select">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>1 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 bit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COM PORT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DATA BITS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP BITS:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PARITY_select">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EVEN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ODD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SPACE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MARK</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PARITY:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="PWM_D_groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>PWM D</string>
    </property>
    <widget class="QCheckBox" name="PWM_D_ON_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PWM_D_duty_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Duty:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SEND_data_button_D">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send data</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PWM_C_groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>PWM C</string>
    </property>
    <widget class="QCheckBox" name="PWM_C_ON_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PWM_C_duty_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Duty:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SEND_data_button_C">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send data</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency (PWM A-D):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PWM_A_D_frq_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_frq_Button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="Period_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Period: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Accuracy_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy: </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
